### PR TITLE
Add Derisking federal field guide

### PR DESCRIPTION
**Pull request by beechnut**
To be merged after #48 

Resolves 18F/TLC-crew#353

### CODE REVIEW

**Comment by beechnut** [comment]
I'm struggling with these checks: locally, my links all pass. I'm also having trouble finding the footnote referenced in the pa11y error.

**Comment by igorkorenfeld** [comment]
The pa11y issue is the link here in the table on the [planning page](https://federalist-864a0936-e148-4df8-8602-6b0992887566.sites.pages.cloud.gov/preview/18f/approaches/mc/add-derisking-fed/derisking/federal-field-guide/planning/#software-development-efforts-should-be-tightly-scoped-to-reduce-risk-and-avoid-overspending). On the current guide the table header is a light grey, but on this one it's a darker grey. 
image
image

The link issue is on the [Deciding what to buy page](https://federalist-864a0936-e148-4df8-8602-6b0992887566.sites.pages.cloud.gov/preview/18f/approaches/mc/add-derisking-fed/derisking/federal-field-guide/deciding-what-to-buy/#recommendation-2). I think we need to pass it through the URL filter, and hopefully it'll work.

File: content/derisking/federal-field-guide/4-deciding-what-to-buy.md [hunk 1]
```diff
@@ -0,0 +1,698 @@
...
+
+T&M allows agencies to save money, too. The contractor can only bill for time spent building a product. If there are delays hiring a new team member or a person goes on extended leave, the government is not paying for that time. T&M provides the opportunity for costs to come in under budget, which never happens with FFP.
+
+A final T&M benefit is that if the contractor is not producing quality software — if their work is inadequate or their skills prove to not fit the work needed — no more work needs be assigned to the contractor, which effectively terminates the contract. There is no work in the product backlog, and the contractor cannot bill their time to the government. There is no need to terminate for convenience or cause.
+
+To be clear, T&M does require more active management than the FFP "set it and forget it" contracts. The government must know what is going on, and review and approve invoices appropriately. Luckily, with agile development, this should already be happening. Read more about this shift in our recommendation on [post-award management](/federal-field-guide/doing-the-work/#post-award-contract-administration-looks-different-in-agile).
```
Comment (igorkorenfeld) [inline content/derisking/federal-field-guide/4-deciding-what-to-buy.md:427]: This is the broken link that probably needs a `| url`

Comment (beechnut) [reply content/derisking/federal-field-guide/4-deciding-what-to-buy.md:427]: Must have been afternoon brain — I could _not_ find this yesterday. Thank you. Fixed.

**Comment by beechnut** [comment]
Hm, it looks like since it's a footnote there's nothing to pass through the URL filter. I suspect the fix here is to port the table style from the current guide.

Perhaps I should close these PRs and open a new combined one, since the style work is going to affect both — thoughts @igorkorenfeld ?

**Comment by igorkorenfeld** [comment]
That makes a lot of sense and I like your approach! Would you mind adding some documentation about the titles to `replatforming.md` so we can reference it later?

**Comment by igorkorenfeld** [comment]
I realized there is a small issue with the new title approach that I didn't catch on Friday. The order of the headers will now be `<h2>` `<h1>` `<h2>` ... which is typically discouraged.

**Comment by igorkorenfeld** [comment]
Updated this PR to work with Matt's implementation of custom titles to keep the descending heading order. I've updated the docs to capture the new updates, incorporating the documentation Matt had written up. I've combined it into the existing doc to keep everything in one place – I didn't want to split out the doc yet, particularly as we are approaching the next phase of the guides work. 

Additionally organized the Javascript assets so that they are placed in guide specific folders, following the pattern we have used for the other assets.

File: _includes/layouts/page.html [hunk 2]
```diff
@@ -9,23 +9,30 @@
...
       </main>
     </div>
   </div>
 </div>
 
-  <script src={{ '/assets/_common/js/current-link.js' | url }}></script>
-{% if script %}
-  <script src="{{ script | url }}"></script>
+{% if tags[0] == 'derisking' %}
+  <script src={{ '/assets/derisking/js/current-link.js' | url }}></script>
```
Comment (echappen) [inline _includes/layouts/page.html:34]: [non-blocking question] Does the Derisking Guide have a unique JS asset, or will other guides need this asset as well?

**Review by echappen — APPROVED** [review]
LGTM!

File: _includes/layouts/page.html (hunk 2 above)
Comment (igorkorenfeld) [reply _includes/layouts/page.html:34]: This one is unique to the De-risking guide, the other guides don't use it.

**Pull request merged**